corehq/apps/accounting/migrations/0072_export_multisort_priv.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.16 on 2023-04-05 06:55
+
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.apps.accounting.models import SoftwarePlanEdition
+from corehq.privileges import EXPORT_MULTISORT
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_export_multisort_priv(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+
+    # EXPORT_MULTISORT are Standard Plan and higher
+    skip_editions = ','.join((
+        SoftwarePlanEdition.PAUSED,
+        SoftwarePlanEdition.COMMUNITY,
+    ))
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        EXPORT_MULTISORT,
+        skip_edition=skip_editions,
+        noinput=True,
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0071_add_billingaccountwebuserhistory'),
+    ]
+
+    operations = [
+        migrations.RunPython(
```

Comment:
Should we not also add the `down` for these migrations? I know the likelyhood of this causing an issue is slim, but having the down function will
1. save whoever is deploying this the trouble of having to manually go and remove this role in the case of disaster
2. not cause any unwanted "artifacts" in the database if we run the `down` migration for some reason

Reply:
I added a down/reverse function to remove the created role and grants

Reply:
Looks good. :+1: For future grooming meetings, we should just ensure that one of the requirements for a GA ticket is to include a `reverse_code` function for if then migration needs to be reverted.

Reply:
I talked with Sravan and it turns out that by adding this reverse function might limit our ability to remove a privilege, if and when it is required in the future. By doing this, we'll just remove the role + grants, but it might be that we want other sensitive things to happen as well. So it is recommended to rather leave this out from these types of migrations and let the person in the future deal with everything.

That said, I'm going to remove this.